extensions/dicom-microscopy/src/components/MicroscopyLabelDialog/MicroscopyLabelDialog.styl

```diff
@@ -0,0 +1,3 @@
+.InputDialog.MicroscopyLabelDialog .content
```

Comment:
can we jsut remove these two .style files since there is no .style in v3, it was in v2. Maybe work it with tailwindcss